composable_prepare_nic_configs.yml

```diff
@@ -34,14 +34,25 @@
 
           - name: set fact rhel8_interfaces
             set_fact:
-              rhel8_interfaces: "{{  rhel8_interfaces|default([]) + [((item.changed == False) | ternary('rhel8_interfaces', ( item.changed == true and item.stdout == 'TN10RT') | ternary('rhel8_interfaces_tn10rt', 'rhel8_interfaces_trtp')))] }}"
-            with_items:
+              rhel8_interfaces: "{{  rhel8_interfaces|default([]) + [((item.0.changed == False) | ternary((item.0.stdout.split('-')[0] == 'f04' and item.2 == 'r640') | ternary ('rhel8_interfaces_f04', 'rhel8_interfaces'), (item.0.changed == true and item.0.stdout == 'TN10RT') | ternary('rhel8_interfaces_tn10rt', 'rhel8_interfaces_trtp')))] }}"
```

Comment:
should be item.1.stdout.split('-')[0] == 'f04'

Reply:
Damn, the penalty for not having an env to test! Appreciate it! Good eyes.